code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -233,6 +233,13 @@ GLOBAL_LIST_EMPTY(pipeimages)
 		var/datum/gas_mixture/env_air = loc.return_air()
 		pressures = int_air.return_pressure() - env_air.return_pressure()
 
+	if(ishuman(user))
+		var/mob/living/carbon/human/H = user
+		var/obj/item/clothing/shoes/magboots/magboot = H.get_item_by_slot(SLOT_SHOES)
+		if(magboot && istype(magboot) && magboot.magpulse)
+			return // Dont send you flying
+		
+
```

Comment:
```suggestion
```

Reply:
im murdering you

Reply:
> im murdering you

added to the evidence I'll leave behind if someone murders me